x-pack/plugin/inference/src/test/java/org/elasticsearch/xpack/inference/services/custom/CustomServiceTests.java

```diff
@@ -807,17 +822,4 @@ public void testChunkedInfer_ChunkingSettingsNotSet() throws IOException {
             assertThat(requestMap.get("input"), is(List.of("a")));
         }
     }
-
-    @Override
-    public InferenceService createInferenceService() {
-        return createService(threadPool, clientManager);
-    }
-
-    @Override
-    protected void assertRerankerWindowSize(RerankingInferenceService rerankingInferenceService) {
```

Comment:
The abstract base class now implements this and calls the test configuration method that the service registers in the constructor.